### PR TITLE
Rewrite chapters on testing

### PR DESCRIPTION
**Pull request by euonymos**
Code's updates are [here](https://github.com/geniusyield/atlas/pull/337).

### CODE REVIEW

File: src/pages/getting-started/testing.mdx [hunk 1]
```diff
@@ -0,0 +1,549 @@
...
+  to prepare a hand-made environment for most intermediary transactions to be run in
+  without running transactions that precede, so mostly it boils down
+  to test the whole transaction flow.
+
+## Overview of unified testing in Atlas
+
+Testing of whole operations (transactions) requires a Cardano ledger to evaluate them
+and keep the state. 
+There are two interchangeable options available in Atlas. 
+We will call them **ledger backends* throughout the rest of the section):
```
Comment (sourabhxyz) [inline src/pages/getting-started/testing.mdx:57]:
```suggestion
We will call them *ledger backends* throughout the rest of the section:
```

File: src/pages/getting-started/testing.mdx [hunk 2]
```diff
@@ -0,0 +1,549 @@
...
+
+## Overview of unified testing in Atlas
+
+Testing of whole operations (transactions) requires a Cardano ledger to evaluate them
+and keep the state. 
+There are two interchangeable options available in Atlas. 
+We will call them *ledger backends* throughout the rest of the section:
+
+  * __[CLB](https://github.com/mlabs-haskell/clb) emulator__
+  (a modern replacement for deprecated PSM library)
```
Comment (sourabhxyz) [inline src/pages/getting-started/testing.mdx:60]:
```suggestion
  (a modern replacement for deprecated [PSM](https://github.com/mlabs-haskell/plutus-simple-model) library)
```

**Review by sourabhxyz — APPROVED** [review]
Thanks @euonymos!

**Review by 4TT1L4 — APPROVED** [review]
LGTM

**Pull request merged**